### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/owncloud-client/commit/d234b5cb6721d9f59e44b52a964f43cae0d47937#commitcomment-56639815

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/owncloud-client/commit/dc7338c826600fe31a05ddd0d6cb548b3f8815cc

Note: this PR does not include the action. If you're interested in running a spell check on every PR and push, that can be offered separately.

### CODE REVIEW

**Comment by TheOneRing** [comment]
Woa sounds great and I'd love a spell checker action.
I will need some time to check those.
Note: Some of the files where auto generated to fixing the spelling in the changelog.md for example doesn't make much sense.
I also saw some unrelated code style changes?

File: src/gui/socketapi/socketuploadjob.cpp [hunk 1]
```diff
@@ -58,11 +58,11 @@ void SocketUploadJob::prepareTag(const AccountPtr &account)
 
         connect(propfindJob, &LsColJob::directoryListingIterated, this, [this](const QString &, const QMap<QString, QString> &data) {
             if (data[QStringLiteral("display-name")] == backupTagNameC()) {
-                _finisedTagId = data[QStringLiteral("id")].toInt();
+                _finishedTagId = data[QStringLiteral("id")].toInt();
             }
         });
         connect(propfindJob, &LsColJob::finishedWithError, this, [this] {
-            fail(tr("Failed to rerieve tags"));
+            fail(tr("Failed to retrieve tags"));
```
Comment (jsoref) [inline src/gui/socketapi/socketuploadjob.cpp:65]: user facing?

File: src/gui/wizard/owncloudadvancedsetuppage.ui [hunk 2]
```diff
@@ -177,7 +177,7 @@
                  </widget>
                 </item>
                 <item>
-                 <widget class="QLabel" name="confTraillingSizeLabel">
+                 <widget class="QLabel" name="confTrailingSizeLabel">
```
Comment (jsoref) [inline src/gui/wizard/owncloudadvancedsetuppage.ui:180]: ux api??

File: src/gui/wizard/owncloudadvancedsetuppage.ui [hunk 3]
```diff
@@ -469,7 +469,7 @@
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;If this box is checked, existing content in the local folder will be erased to start a clean sync from the server.&lt;/p&gt;&lt;p&gt;Do not check this if the local content should be uploaded to the servers folder.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
               <property name="text">
-               <string>Start a &amp;clean sync (Crreate a backup and erases the local folder!)</string>
+               <string>Start a &amp;clean sync (Create a backup and erases the local folder!)</string>
```
Comment (jsoref) [inline src/gui/wizard/owncloudadvancedsetuppage.ui:472]: user facing?

File: src/libsync/abstractnetworkjob.h [hunk 4]
```diff
@@ -80,7 +80,7 @@ class OWNCLOUDSYNC_EXPORT AbstractNetworkJob : public QObject
      * This function reads the body of the reply and parses out the
      * error information, if possible.
      *
-     * \a body is optinally filled with the reply body.
+     * \a body is optionally filled with the reply body.
```
Comment (jsoref) [inline src/libsync/abstractnetworkjob.h:83]: api consumer facing?

File: src/libsync/creds/credentialmanager.cpp [hunk 5]
```diff
@@ -138,7 +138,7 @@ QStringList CredentialManager::knownKeys(const QString &group) const
 
 QSettings *CredentialManager::credentialsList() const
 {
-    // delayed init as scope requires a fully inizialised acc
+    // delayed init as scope requires a fully initialised acc
```
Comment (jsoref) [inline src/libsync/creds/credentialmanager.cpp:141]: I personally speak en-US, but for this repository, it doesn't look like I check for en-GB/en-US. And in general, I try not to meddle w/ locale preferences. That said, he `z` at the beginning gave me momentary pause.

File: test/csync/std_tests/check_std_c_jhash.c [hunk 6]
```diff
@@ -9,169 +9,181 @@
...
-    z=0;
-    for (i=0; i<hlen; ++i) {  /*----------------------- for each input byte, */
-      for (j=0; j<8; ++j) { /*------------------------ for each input bit, */
-        for (m=1; m<8; ++m) { /*------------ for serveral possible initvals, */
-          for (l=0; l<HASHSTATE; ++l) e[l]=f[l]=g[l]=h[l]=x[l]=y[l]=~((uint32_t)0);
+    for (hlen = 0; hlen < MAXLEN; ++hlen) {
+        z = 0;
+        for (i = 0; i < hlen; ++i) { /*----------------------- for each input byte, */
+            for (j = 0; j < 8; ++j) { /*------------------------ for each input bit, */
+                for (m = 1; m < 8; ++m) { /*------------ for several possible initvals, */
```
Comment (jsoref) [inline test/csync/std_tests/check_std_c_jhash.c:30]: `several`

File: test/testallfilesdeleted.cpp [hunk 7]
```diff
@@ -252,7 +252,7 @@ private slots:
         QVERIFY(fakeFolder.syncOnce());
         QCOMPARE(fingerprintRequests, 2);
         auto currentState = fakeFolder.currentLocalState();
-        // Altough the local file is kept as a conflict, the server file is downloaded
+        // Although the local file is kept as a conflict, the server file is downloaded
```
Comment (jsoref) [inline test/testallfilesdeleted.cpp:255]: `Although`

File: test/testallfilesdeleted.cpp [hunk 8]
```diff
@@ -315,15 +317,16 @@ private slots:
...
         fakeFolder.syncEngine().journal()->setSelectiveSyncList(SyncJournalDb::SelectiveSyncBlackList,
-            QStringList() << "A/" << "B/" << "C/" << "S/");
+            QStringList() << "A/"
+                          << "B/"
+                          << "C/"
+                          << "S/");
 
         QVERIFY(fakeFolder.syncOnce());
-        QCOMPARE(fakeFolder.currentLocalState(), FileInfo{}); // all files should be one localy
+        QCOMPARE(fakeFolder.currentLocalState(), FileInfo {}); // all files should be one locally
```
Comment (jsoref) [inline test/testallfilesdeleted.cpp:326]: `locally`

File: test/testcookies.cpp [hunk 9]
```diff
@@ -27,7 +27,7 @@ private slots:
         jar.setAllCookies(cookies);
         QCOMPARE(cookies, jar.allCookies());
         QVERIFY(jar.save(tmp.filePath("test.db")));
-        // ensure we are able to create a cookie jar in a non exisitning folder (mkdir)
+        // ensure we are able to create a cookie jar in a non existing folder (mkdir)
```
Comment (jsoref) [inline test/testcookies.cpp:30]:
```suggestion
        // ensure we are able to create a cookie jar in a nonexistent folder (mkdir)
```

File: test/testdownload.cpp [hunk 10]
```diff
@@ -109,24 +110,26 @@ private slots:
...
         QTimer::singleShot(10000, &fakeFolder.syncEngine(), [&]() { timedOut = true; fakeFolder.syncEngine().abort(); });
-        QVERIFY(!fakeFolder.syncOnce());  // Fail because A/broken
+        QVERIFY(!fakeFolder.syncOnce()); // Fail because A/broken
         QVERIFY(!timedOut);
         QCOMPARE(getItem(completeSpy, "A/broken")->_status, SyncFileItem::NormalError);
         QVERIFY(getItem(completeSpy, "A/broken")->_errorString.contains(serverMessage));
     }
 
-    void serverMaintenence() {
+    void serverMaintenance()
```
Comment (jsoref) [inline test/testdownload.cpp:128]: `serverMaintenance`

File: test/testexcludedfiles.cpp [hunk 11]
```diff
@@ -111,7 +111,7 @@ private slots:
         QVERIFY(check_isExcluded(QStringLiteral("/a/foo (conflicted copy bar)"), keepHidden));
         QVERIFY(check_isExcluded(QStringLiteral("/a/.b"), excludeHidden));
 
-        // test non exisitng folder
+        // test non existing folder
```
Comment (jsoref) [inline test/testexcludedfiles.cpp:114]:
```suggestion
        // test nonexistent folder
```

**Review by jsoref — COMMENTED** [review]
This correction pass was almost entirely made by Google Sheets.

I sometimes look for non-existent variants and similar, but didn't here.

I'm happy to make fixes (but probably won't have time for a while).

File: src/common/syncjournaldb.cpp [hunk 12]
```diff
@@ -17,21 +17,21 @@
...
+#include <QElapsedTimer>
 #include <QFile>
 #include <QLoggingCategory>
 #include <QStringList>
-#include <QElapsedTimer>
 #include <QUrl>
-#include <QDir>
-#include <sqlite3.h>
 #include <cstring>
+#include <sqlite3.h>
```
[the remark this answers is not in the document]
Comment (jsoref) [reply src/common/syncjournaldb.cpp:27]: @TheOneRing re https://github.com/owncloud/client/pull/9044#issuecomment-921629336 this is:
https://github.com/jsoref/owncloud-client/commit/51b6bb7eda59ad346336f5010ff867f2fe1171ad

I'd encourage you to merge that commit outside of this PR, since it really doesn't belong, but the linter failed and it was way too messy for me to handle it any other way here. -- And the only way to show it mattered was to compose this PR.

**Comment by jsoref** [comment]
Fwiw, this is what I used to ignore files:
https://github.com/jsoref/owncloud-client/blob/spell-check/.github/actions/spelling/excludes.txt

Files that were ignored weren't checked for typos, and as long as I didn't make any mistakes, ignored files weren't changed either (beyond the linter changes which are really an entirely unrelated kettle of fish).

I specifically ignored `^changelog/[^/]*_` for this repository (but not some of the others), this may have been the wrong choice. It isn't terribly hard for me to unignore items, either. The spell checker uses fairly naive heuristics to suggest exclusions, and I generalize based on my experience w/ things, but I'm human and the spell-checker was written by me, so both stages can make mistakes.

If you can identify other files you don't want changed, it's trivial for me to add them to excludes and to drop their changes from this PR (or its relatives for the other owncloud repositories).

If there's an upstream for typos, feel free to identify them, I have an imaginary queue of things to check, and will happily add items to the queue, although I make no promises about when i'll get to items in it 😺.

File: resources/font-awesome/update_color.ps1 [hunk 13]
```diff
@@ -6,7 +6,7 @@ $COLOR=$Args[0]
 Write-Host "Color:" $COLOR
 if (-not $COLOR) {
     Write-Host "Please specify color"
-    Write-Host "update_corlor.ps1 `"#B5B6BB`""
+    Write-Host "update_color.ps1 `"#B5B6BB`""
```
Comment (jsoref) [inline resources/font-awesome/update_color.ps1:9]: This makes the output match the filename...

File: test/testsyncvirtualfiles.cpp [hunk 14]
```diff
@@ -142,7 +142,7 @@ private slots:
         QCOMPARE(dbRecord(fakeFolder, "A/a1" DVSUFFIX)._fileSize, 65);
         cleanup();
 
-        // If the local virtual file file is removed, it'll just be recreated
+        // If the local virtual file is removed, it'll just be recreated
```
Comment (jsoref) [inline test/testsyncvirtualfiles.cpp:145]: I've recently added some code to detect word duplication. I can certainly suppress it if it's annoying, but on average it's pretty accurate (and a handful of additional patterns are sufficient to cover the edge cases where it's wrong).

File: test/testpermissions.cpp [hunk 15]
```diff
@@ -484,22 +484,22 @@ private slots:
         rm.mkdir("changeonly");
         rm.mkdir("changeonly/sub1");
         rm.insert("changeonly/sub1/file1");
-        rm.insert("changeonly/sub1/filetorname1a");
-        rm.insert("changeonly/sub1/filetorname1z");
+        rm.insert("changeonly/sub1/filetorename1a");
+        rm.insert("changeonly/sub1/filetorename1z");
```
Comment (jsoref) [inline test/testpermissions.cpp:488]: I didn't see any particular reason to omit the `e` from `rename`

File: test/testallfilesdeleted.cpp [hunk 16]
```diff
@@ -184,20 +185,19 @@ private slots:
         // Now, aboutToRemoveAllFiles with down as a direction
         QVERIFY(fakeFolder.syncOnce());
         QCOMPARE(aboutToRemoveAllFilesCalled, 1);
-
     }
 
-    void testDataFingetPrint_data()
+    void testDataFingerPrint_data()
```
Comment (jsoref) [inline test/testallfilesdeleted.cpp:190]: `finger`

File: test/gui/tst_checkAlltabs/test.feature [hunk 17]
```diff
@@ -2,7 +2,7 @@ Feature: Visually check all tabs
 
     As a user
     I want to visually check all tabs in client
-    So that I can performe all the actions related to client
+    So that I can perform all the actions related to client
 
 
     Scenario: Tabs in toolbar looks correct
```
Comment (jsoref) [inline test/gui/tst_checkAlltabs/test.feature:8]: The folder below is gaining an `e` in `delete`:

File: test/gui/suite.conf [hunk 18]
```diff
@@ -4,6 +4,6 @@ HOOK_SUB_PROCESSES=false
 IMPLICITAUTSTART=0
 LANGUAGE=Python
 OBJECTMAPSTYLE=script
-TEST_CASES=tst_addAccount tst_sharing tst_syncing tst_loginLogout tst_removeAccountConnection tst_checkAlltabs tst_vfs tst_deletFilesFolders tst_editFiles
+TEST_CASES=tst_addAccount tst_sharing tst_syncing tst_loginLogout tst_removeAccountConnection tst_checkAlltabs tst_vfs tst_deleteFilesFolders tst_editFiles
```
Comment (jsoref) [inline test/gui/suite.conf:7]: The test case is gaining an `e` in `delete` (see renames below)

**Comment by ownclouders** [comment]
Results for <strong>GUI-tests</strong> https://drone.owncloud.com/owncloud/client/11261/5/1
:boom: The GUI tests failed.
GUI Logs: (https://cache.owncloud.com/public/owncloud/client/11261/guiReportUpload/index.html)
Server Logs: (https://cache.owncloud.com/public/owncloud/client/11261/guiReportUpload/serverlog.log)

File: src/libsync/theme.h [hunk 19]
```diff
@@ -73,7 +73,7 @@ class OWNCLOUDSYNC_EXPORT Theme : public QObject
      *
      * By default, the name is derived from the APPLICATION_SHORTNAME
      * cmake variable, and should be the same. This method is only
-     * reimplementable for legacy reasons.
+     * overridable for legacy reasons.
```
Comment (jsoref) [inline src/libsync/theme.h:76]: Notable

File: shell_integration/windows/OCOverlays/OCOverlay.cpp [hunk 20]
```diff
@@ -101,8 +101,8 @@ IFACEMETHODIMP_(ULONG) OCOverlay::Release()
 
 IFACEMETHODIMP OCOverlay::GetPriority(int *pPriority)
 {
-    // this defines which handler has prededence, so
-    // we order this in terms of likelyhood
+    // this defines which handler has precedence, so
+    // we order this in terms of likelihood
     switch (_state) {
     case State_OK:
         *pPriority = 0; break;
```
Comment (jsoref) [inline shell_integration/windows/OCOverlays/OCOverlay.cpp:108]: The below file is being renamed from `deprACtion` to `depreCAtion`:

**Review by jsoref — COMMENTED** [review]
At the risk of losing GitHub's fancy review features, you can use:
https://github.com/owncloud/client/compare/6cf97832cfe536d49d44155a65e009c512236b65...4a818debf35109e4521049b480d79d2cff3a3bd4 to confirm that the only commit that's changing style is https://github.com/owncloud/client/commit/6cf97832cfe536d49d44155a65e009c512236b65 -- it's necessary because the linter screams at me if I don't do it (as noted earlier).

Note: the repository has lots of files that would fail the linter, it's really an obstacle for drive-by-contributions. If this were my project, I'd apply the linter to all files at once so that any new change wouldn't be forced to deal w/ files that aren't linter-stable.

**Comment by ownclouders** [comment]
Results for <strong>GUI-tests</strong> https://drone.owncloud.com/owncloud/client/11728/6/1

:boom: The GUI tests failed.

GUI Logs: https://cache.owncloud.com/public/owncloud/client/11728/guiReportUpload/index.html

<details>

Server Logs: https://cache.owncloud.com/public/owncloud/client/11728/guiReportUpload/serverlog.log

Screenshots:
- [Adding_account_with_self_signed_certificate_for_the_first_time.png](https://cache.owncloud.com/public/owncloud/client/11728/guiReportUpload/screenshots/Adding_account_with_self_signed_certificate_for_the_first_time.png)
- [Adding_account_with_wrong_credentials.png](https://cache.owncloud.com/public/owncloud/client/11728/guiReportUpload/screenshots/Adding_account_with_wrong_credentials.png)
- [Adding_multiple_accounts.png](https://cache.owncloud.com/public/owncloud/client/11728/guiReportUpload/screenshots/Adding_multiple_accounts.png)
- [Adding_normal_Account.png](https://cache.owncloud.com/public/owncloud/client/11728/guiReportUpload/screenshots/Adding_normal_Account.png)
- [Both_original_and_copied_folders_can_be_synced.png](https://cache.owncloud.com/public/owncloud/client/11728/guiReportUpload/screenshots/Both_original_and_copied_folders_can_be_synced.png)
- [Connect_account_with_manual_sync_folder_option.png](https://cache.owncloud.com/public/owncloud/client/11728/guiReportUpload/screenshots/Connect_account_with_manual_sync_folder_option.png)
- [Delete_a_file.png](https://cache.owncloud.com/public/owncloud/client/11728/guiReportUpload/screenshots/Delete_a_file.png)
- [Delete_a_folder.png](https://cache.owncloud.com/public/owncloud/client/11728/guiReportUpload/screenshots/Delete_a_folder.png)
- [Disable_VFS.png](https://cache.owncloud.com/public/owncloud/client/11728/guiReportUpload/screenshots/Disable_VFS.png)
- [File_with_long_name_(>228_characters)_is_blacklisted.png](https://cache.owncloud.com/public/owncloud/client/11728/guiReportUpload/screenshots/File_with_long_name_(>228_characters)_is_blacklisted.png)
- [File_with_long_name_can_be_synced.png](https://cache.owncloud.com/public/owncloud/client/11728/guiReportUpload/screenshots/File_with_long_name_can_be_synced.png)
- [File_with_spaces_in_the_name_can_sync.png](https://cache.owncloud.com/public/owncloud/client/11728/guiReportUpload/screenshots/File_with_spaces_in_the_name_can_sync.png)
- [Filenames_that_are_rejected_by_the_server_are_reported.png](https://cache.owncloud.com/public/owncloud/client/11728/guiReportUpload/screenshots/Filenames_that_are_rejected_by_the_server_are_reported.png)
- [Invalid_system_names_are_synced_in_linux.png](https://cache.owncloud.com/public/owncloud/client/11728/guiReportUpload/screenshots/Invalid_system_names_are_synced_in_linux.png)
- [Many_subfolders_can_be_synced.png](https://cache.owncloud.com/public/owncloud/client/11728/guiReportUpload/screenshots/Many_subfolders_can_be_synced.png)
- [Modify_orignal_content_of_a_file_with_special_character.png](https://cache.owncloud.com/public/owncloud/client/11728/guiReportUpload/screenshots/Modify_orignal_content_of_a_file_with_special_character.png)
- [Progress_indicator_should_not_be_visible_after_unselecting_the_password_protection_checkbox_while_sharing_through_public_link.png](https://cache.owncloud.com/public/owncloud/client/11728/guiReportUpload/screenshots/Progress_indicator_should_not_be_visible_after_unselecting_the_password_protection_checkbox_while_sharing_through_public_link.png)
- [Sync_all_is_selected_by_default.png](https://cache.owncloud.com/public/owncloud/client/11728/guiReportUpload/screenshots/Sync_all_is_selected_by_default.png)
- [Sync_only_one_folder_from_the_server.png](https://cache.owncloud.com/public/owncloud/client/11728/guiReportUpload/screenshots/Sync_only_one_folder_from_the_server.png)
- [Syncing_a_file_from_the_server_and_creating_a_conflict.png](https://cache.owncloud.com/public/owncloud/client/11728/guiReportUpload/screenshots/Syncing_a_file_from_the_server_and_creating_a_conflict.png)
- [Syncing_a_file_to_the_server.png](https://cache.owncloud.com/public/owncloud/client/11728/guiReportUpload/screenshots/Syncing_a_file_to_the_server.png)
- [Syncing_a_folder_to_the_server.png](https://cache.owncloud.com/public/owncloud/client/11728/guiReportUpload/screenshots/Syncing_a_folder_to_the_server.png)
- [Syncing_all_files_and_folders_from_the_server.png](https://cache.owncloud.com/public/owncloud/client/11728/guiReportUpload/screenshots/Syncing_all_files_and_folders_from_the_server.png)
- [Syncing_file_of_1_GB_size.png](https://cache.owncloud.com/public/owncloud/client/11728/guiReportUpload/screenshots/Syncing_file_of_1_GB_size.png)
- [Syncing_folders_each_having_500_files.png](https://cache.owncloud.com/public/owncloud/client/11728/guiReportUpload/screenshots/Syncing_folders_each_having_500_files.png)
- [VFS_is_disabled_by_default.png](https://cache.owncloud.com/public/owncloud/client/11728/guiReportUpload/screenshots/VFS_is_disabled_by_default.png)
- [Verify_one_empty_folder_with_a_length_longer_than_the_allowed_limit_will_not_be_synced.png](https://cache.owncloud.com/public/owncloud/client/11728/guiReportUpload/screenshots/Verify_one_empty_folder_with_a_length_longer_than_the_allowed_limit_will_not_be_synced.png)
- [Verify_pre_existing_folders_in_local_(Desktop_client)_are_copied_over_to_the_server.png](https://cache.owncloud.com/public/owncloud/client/11728/guiReportUpload/screenshots/Verify_pre_existing_folders_in_local_(Desktop_client)_are_copied_over_to_the_server.png)
- [Verify_that_you_can_create_a_subfolder_with_long_name.png](https://cache.owncloud.com/public/owncloud/client/11728/guiReportUpload/screenshots/Verify_that_you_can_create_a_subfolder_with_long_name.png)
- [change_collaborator_permissions_of_a_file_&_folder.png](https://cache.owncloud.com/public/owncloud/client/11728/guiReportUpload/screenshots/change_collaborator_permissions_of_a_file_&_folder.png)
- [collaborators_are_listed_in_chronological_order.png](https://cache.owncloud.com/public/owncloud/client/11728/guiReportUpload/screenshots/collaborators_are_listed_in_chronological_order.png)
- [logging_out.png](https://cache.owncloud.com/public/owncloud/client/11728/guiReportUpload/screenshots/logging_out.png)
- [remove_an_account_connection.png](https://cache.owncloud.com/public/owncloud/client/11728/guiReportUpload/screenshots/remove_an_account_connection.png)
- [remove_the_only_account_connection.png](https://cache.owncloud.com/public/owncloud/client/11728/guiReportUpload/screenshots/remove_the_only_account_connection.png)
- [search_for_users_with_minimum_autocomplete_characters.png](https://cache.owncloud.com/public/owncloud/client/11728/guiReportUpload/screenshots/search_for_users_with_minimum_autocomplete_characters.png)
- [share_a_file_with_many_users.png](https://cache.owncloud.com/public/owncloud/client/11728/guiReportUpload/screenshots/share_a_file_with_many_users.png)
- [share_file_and_folder_to_a_group.png](https://cache.owncloud.com/public/owncloud/client/11728/guiReportUpload/screenshots/share_file_and_folder_to_a_group.png)
- [sharee_creates_a_file_and_a_folder_inside_a_shared_folder.png](https://cache.owncloud.com/public/owncloud/client/11728/guiReportUpload/screenshots/sharee_creates_a_file_and_a_folder_inside_a_shared_folder.png)
- [sharee_deletes_a_file_and_folder_shared_by_sharer.png](https://cache.owncloud.com/public/owncloud/client/11728/guiReportUpload/screenshots/sharee_deletes_a_file_and_folder_shared_by_sharer.png)
- [sharee_edits_content_of_files_shared_by_sharer.png](https://cache.owncloud.com/public/owncloud/client/11728/guiReportUpload/screenshots/sharee_edits_content_of_files_shared_by_sharer.png)
- [sharee_edits_shared_files_and_again_try_to_edit_after_write_permission_is_revoked.png](https://cache.owncloud.com/public/owncloud/client/11728/guiReportUpload/screenshots/sharee_edits_shared_files_and_again_try_to_edit_after_write_permission_is_revoked.png)
- [sharee_renames_the_shared_file_and_folder.png](https://cache.owncloud.com/public/owncloud/client/11728/guiReportUpload/screenshots/sharee_renames_the_shared_file_and_folder.png)
- [sharee_tries_to_create_a_file_and_a_folder_inside_a_shared_folder_without_write_permission.png](https://cache.owncloud.com/public/owncloud/client/11728/guiReportUpload/screenshots/sharee_tries_to_create_a_file_and_a_folder_inside_a_shared_folder_without_write_permission.png)
- [sharee_tries_to_delete_shared_file_and_folder_without_permissions.png](https://cache.owncloud.com/public/owncloud/client/11728/guiReportUpload/screenshots/sharee_tries_to_delete_shared_file_and_folder_without_permissions.png)
- [sharee_tries_to_edit_content_of_files_shared_without_write_permission.png](https://cache.owncloud.com/public/owncloud/client/11728/guiReportUpload/screenshots/sharee_tries_to_edit_content_of_files_shared_without_write_permission.png)
- [sharing_a_folder_by_public_link_with_"Uploader"_role_and_check_if_file_can_be_downloaded.png](https://cache.owncloud.com/public/owncloud/client/11728/guiReportUpload/screenshots/sharing_a_folder_by_public_link_with_"Uploader"_role_and_check_if_file_can_be_downloaded.png)
- [sharing_file_and_folder_with_user_who_has_some_other_shares.png](https://cache.owncloud.com/public/owncloud/client/11728/guiReportUpload/screenshots/sharing_file_and_folder_with_user_who_has_some_other_shares.png)
- [sharing_of_a_file_by_public_link_and_deleting_the_link.png](https://cache.owncloud.com/public/owncloud/client/11728/guiReportUpload/screenshots/sharing_of_a_file_by_public_link_and_deleting_the_link.png)
- [sharing_of_a_file_by_public_link_with_password_and_changing_the_password.png](https://cache.owncloud.com/public/owncloud/client/11728/guiReportUpload/screenshots/sharing_of_a_file_by_public_link_with_password_and_changing_the_password.png)
- [simple_sharing_of_a_file_by_public_link_with_default_expiration_date.png](https://cache.owncloud.com/public/owncloud/client/11728/guiReportUpload/screenshots/simple_sharing_of_a_file_by_public_link_with_default_expiration_date.png)
- [simple_sharing_of_file_and_folder_by_public_link_with_password.png](https://cache.owncloud.com/public/owncloud/client/11728/guiReportUpload/screenshots/simple_sharing_of_file_and_folder_by_public_link_with_password.png)
- [simple_sharing_of_file_and_folder_by_public_link_without_password.png](https://cache.owncloud.com/public/owncloud/client/11728/guiReportUpload/screenshots/simple_sharing_of_file_and_folder_by_public_link_without_password.png)
- [simple_sharing_of_folder_by_public_link_with_different_roles.png](https://cache.owncloud.com/public/owncloud/client/11728/guiReportUpload/screenshots/simple_sharing_of_folder_by_public_link_with_different_roles.png)
- [simple_sharing_with_user.png](https://cache.owncloud.com/public/owncloud/client/11728/guiReportUpload/screenshots/simple_sharing_with_user.png)
- [sort_folders_list_by_name_and_size.png](https://cache.owncloud.com/public/owncloud/client/11728/guiReportUpload/screenshots/sort_folders_list_by_name_and_size.png)
- [unshare_a_reshared_file.png](https://cache.owncloud.com/public/owncloud/client/11728/guiReportUpload/screenshots/unshare_a_reshared_file.png)
- [unshare_a_shared_file_and_folder.png](https://cache.owncloud.com/public/owncloud/client/11728/guiReportUpload/screenshots/unshare_a_shared_file_and_folder.png)
- [various_types_of_files_can_be_synced_from_client_to_server.png](https://cache.owncloud.com/public/owncloud/client/11728/guiReportUpload/screenshots/various_types_of_files_can_be_synced_from_client_to_server.png)
- [various_types_of_files_can_be_synced_from_server_to_client.png](https://cache.owncloud.com/public/owncloud/client/11728/guiReportUpload/screenshots/various_types_of_files_can_be_synced_from_server_to_client.png)

</details>

**Comment by DeepDiver1975** [comment]
outdated ...

**Comment by jsoref** [comment]
Happy to update it if someone would look at it... it doesn't take me much time

**Comment by jsoref** [comment]
I guess I need to figure out how to update tests...
